components/CMakeLists.txt

```diff
@@ -127,16 +119,14 @@ list(APPEND CMAKE_MODULE_PATH ${CIMEROOT}/CIME/non_py/src/CMake)
 
 set(CMAKE_VERBOSE_MAKEFILE TRUE)
 
-if(USE_CUDA)
-  enable_language(CUDA)
-endif()
-
 # Scream manages its own flags
 build_eamxx()
 
 # We do want CMAKE_BUILD_TYPE to be set, but we do NOT want CMake to
 # decide what optimization flags to append, based on build type,
 # for components who rely on CIME for build flags, so make all the following empty.
+# JGF: I think we can remove this once proper CMake names are being used in the
```

Comment:
Agreed. I think the mach files should set these variables to some reasonable values.